pysnippets/Scrubs/pipeline.py

```
@@ -177,7 +177,7 @@ def aggregate_df(df, df_y, cols_flags):
             'y': conversion_measure
         })
         return pd.concat([s1, s2])
-    except:
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)